source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/ray_caster.py

```diff
@@ -48,14 +48,6 @@ class RayCaster(SensorBase):
 
     cfg: RayCasterCfg
     """The configuration parameters."""
-    meshes: ClassVar[dict[str, wp.Mesh]] = {}
```

Comment:
Why was this moved from class variable to instance variable? I thought the reasoning was to not recreate warp meshes if they exist in memory?

Reply:
@pascal-roth for vis.

Reply:
The reason is they do not get properly deallocated when the env is closed. So when you loop through environments (for example in a grid search) it might never create a new mesh.